src/Tribe/Admin/Settings.php

```diff
@@ -202,6 +202,7 @@ public function get_tec_events_menu_slug() {
 	 * Adds the menu and pages for The Events Calendar.
 	 *
 	 * @since 5.15.0
+	 * @since TBD Added 'tec_redirect_guided_time_setup' to redirect users to the Guided Setup.
```

Comment:
```suggestion
	 * @since TBD Added 'tec_admin_redirect_guided_time_setup' to redirect users to the Guided Setup.
```
See comment for line 226.